Makefile

```diff
@@ -1,10 +1,37 @@
 .PHONY: build_macos
 build_macos:
 	@echo "Building Macos..."
-	gogio -icon=./assets/appicon.png -target=macos -arch=amd64 -o ./dist/amd64/Chapar.app .
-	gogio -icon=./assets/appicon.png -target=macos -arch=arm64 -o ./dist/arm64/Chapar.app .
+	gogio -icon=./build/appicon.png -target=macos -arch=amd64 -o ./dist/amd64/Chapar.app .
+	gogio -icon=./build/appicon.png -target=macos -arch=arm64 -o ./dist/arm64/Chapar.app .
 	codesign --force --deep --sign - ./dist/amd64/Chapar.app
 	codesign --force --deep --sign - ./dist/arm64/Chapar.app
+	tar -cJf ./dist/Chapar_macos_amd64.tar.xz --directory=./dist/amd64 Chapar.app
+	tar -cJf ./dist/Chapar_macos_arm64.tar.xz --directory=./dist/arm64 Chapar.app
+	rm -rf ./dist/amd64
+	rm -rf ./dist/arm64
+
+.PHONY: build_windows
+build_windows:
+	@echo "Building Windows..."
+	gogio -icon=build\appicon.png -buildmode=archive -target=windows -arch=amd64 -o dist\amd64\Chapar.exe .
+	gogio -icon=build\appicon.png -buildmode=archive -target=windows -arch=386 -o dist\i386\Chapar.exe .
+	rm *.syso
+	zip -r -j dist\Chapar_windows_amd64.zip dist\amd64\Chapar.exe
+	zip -r -j dist\Chapar_windows_i386.zip dist\i386\Chapar.exe
+	rm -rf dist\amd64
+	rm -rf dist\i386
+
+.PHONY: build_linux
+build_linux:
+	@echo "Building Linux amd64..."
+	CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -o ./dist/amd64/chapar .
+	cp ./build/install-linux.sh ./dist/amd64
+	cp ./build/appicon.png ./dist/amd64
+	cp ./LICENSE ./dist/amd64
+	cp -r ./build/desktop-assets ./dist/amd64
+	tar -cJf ./dist/Chapar_linux_amd64.tar.xz --directory=./dist/amd64 desktop-assets install-linux.sh appicon.png ./LICENSE
+	rm -rf ./dist/amd64
+
 
 .PHONY: build
 build: build_macos
```

Comment:
This is strange (even if not related to the current change)

calling make build on Linux or Windows will call the target to build macOS version

Reply:
This default one is old I’ll remove it